atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -416,7 +416,10 @@ def get_digests(self):
             for image in self.workflow.tag_conf.images:
                 image_str = image.to_str()
                 if image_str in registry.digests:
-                    digest = registry.digests[image_str]
+                    # Must use schema v1 digest because the content will
+                    # be served by pulp/crane, which does not support schema
+                    # v2 digest
+                    digest = registry.digests[image_str]['v1']
```

Comment:
Thinking about this -- won't Fedora be using Koji but not Pulp? So actually we probably need to select the correct schema version here.